topics/galaxy-interface/tutorials/workflow-fairification/tutorial.md

```diff
@@ -0,0 +1,668 @@
+> > Teams | *Not supported in Galaxy workflow interface* | 
+> > License | License | MIT License
+> > Sharing | *Not supported in Galaxy workflow interface* | 
+> > Tags | Tags | `sequence-analysis`, `mapping`
+> > Creators | Creator | 
+> >
+> {: .solution}
+{: .question}
+
+Let's annotate the workflow
+
+> <hands-on-title>Add metadata to the workflow</hands-on-title>
+>
+> 1. Add a proper title in **Name**
+> 2. Add a workflow description in **Annotation**
+> 3. Select a license in **License**
+>
+>    > <comment-title></comment-title>
+>    > The most appropriate **license** are **MIT** or **GNU Affero** (if you want a [copyleft](https://en.wikipedia.org/wiki/Copyleft)).
+>    {: .comment}
+>
+> 4. Add creators in **Creator** with their name and a unique identifier (typically an orcid.org ID)
+>
+>    > <comment-title></comment-title>
+>    > Do not forget to click on **Save**
+>    {: .comment}
+>
+> 5. Add tags in **Tags**
+> 6. Save the workflow
+{: .hands_on}
+
+Tip 2 is now fulfilled. We can now move toward the other left tips:
+
+- Tip 1 (**Findability**): *"Register the workflow."*
+- Tip 3 (**Accessibility**): *"Make source code available in a public code repository."*
+- Tip 4 (**Accessibility**): *"Provide example input data and results along with the workflow."*
+- Tip 10 (**Reusability**): *"Provide clear and concise workflow documentation."*
+
+   The workflow annotation provides a short and concise description of the workflow. So Tip 10 is partially fullfilled. In addition to the workflow annotation, each step could annotated so users could have an idea about the purpose of the steps
+
+   > <details-title>Add step descriptions to fulfill Tip 10</details-title>
+   > 
+   > Let's add label and annotation to each step.
+   >
+   > > <hands-on-title>Add FastQC label and step annotation</hands-on-title>
+   > >
+   > > 1. Click on **FastQC**
+   > > 2. Fill in **Label** with `Quality control` in the right panel
+   > > 3. Fill in **Step Annotation** with `This step uses FastQC to generate statistics of raw reads quality including basic statistics, per base sequence quality, per sequence quality scores, adapter content, etc.`
+   > > 4. Save the workflow
+   > {: .hands_on}
+   > 
+   > We can now do the same for MultiQC.
+   {: .details}
+
+To achive Tip 1 and register the workflow, we will first need to fulfill Tips 3 and 4. Before, we need to be sure that the workflow adheres to best practices for Galaxy workflows.
+
+# Adhere to best practices for Galaxy workflows
+
+There are a number of things the user interface of Galaxy will allow that are not considered best practices because they make the workflow harder to test, use within subworkflows and invocation reports, and consume via the API. It is easier to use workflows in all of these contexts if they stick to the [best practices developed by the commmunity](https://planemo.readthedocs.io/en/latest/best_practices_workflows.html).
```

Comment:
```suggestion
Galaxy has a build-in Wizard to check against [community developed best practice workflows](https://planemo.readthedocs.io/en/latest/best_practices_workflows.html). It will recommend to use a License, add Authors etc but also ensures that workflows are easy to test, that they are usable as subworkflows and invocation reports, and can be consumed easily via an API. The reusablility will be greatly enhanced if you stick to those recommendations.
```